Source/player.h

```diff
@@ -33,6 +33,7 @@ namespace devilution {
 #define MAX_SPELL_LEVEL 15
 #define PLR_NAME_LEN 32
 
+static constexpr size_t NumHotkeys = 16;

```

Comment:
`static` in this context doesn't do anything.
```suggestion
constexpr size_t NumHotkeys = 16;
```